src/einsteinpy/coordinates/core.py

```diff
@@ -22,27 +22,36 @@ def __init__(self, x, y, z):
         self.y = y
         self.z = z
 
-    def norm(self):
+    def __repr__(self):
+        return "Cartesian x: {}, y: {}, z: {}".format(self.x, self.y, self.z)
+
+    def __str__(self):
```

Comment:
@shreyasbapat coverage is not up to the mark because `__repr__` and `__str__` is still untested. How to test them? Suggestions?

Reply:
Done ! Redirected STDOUT to a variable.